mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfigurationUtils.java

```diff
@@ -29,6 +30,7 @@ public static <T extends WorkerConfiguration> T frmProperties(Properties propert
             properties);
         configurationObjectFactory.addCoercible(new MetricsCoercer(properties));
         configurationObjectFactory.addCoercible(new PluginCoercible<>(MetricsCollector.class, properties));
+        configurationObjectFactory.addCoercible(new PluginCoercible<>(AvailabilityZoneUtils.class, properties));
```

Comment:
can we avoid using the class instance injection here? We've been trying to remove these class instance injection from the the config and keep worker config as a simple pojo and eventually move to use a new config framework.